feat: Allow specific indexers in EntityIndexerRegistry (#8929)
shopware/platform: changelog/_unreleased/2025-04-12-entityindexerregistry-extension-indexer-only.md

---
title: Add possibility to the context to define which specific indexer should run
author: Niel Duysters
author_email: niel.duysters@meteor.be
author_github: @nielduysters
---
# Core
* Added `EntityIndexerRegistry::EXTENSION_INDEXER_ONLY` to the `EntityIndexerRegistry` to allow specific indexers.

# Upgrade Information

> ## Add possibility to specify indexer in tontext

Reviewer: ```suggestion
## Add possibility to specify indexer in context
```

> < Add some information and how to use it here>

Reviewer: of course you need to fill that with life :grimacing: that was just a placeholder from my side

---

> title: Add EntityIndexerRegistry::EXTENSION_INDEXER_ONLY

Reviewer: ```suggestion
title: Add possibility to the context to define which specific indexer should run
```

> issue: N/A

Reviewer: ```suggestion
```

author: Niel Duysters
author_email: niel.duysters@meteor.be
author_github: @nielduysters
---
# Core

> * Added `EntityIndexerRegistry::EXTENSION_INDEXER_ONLY` to the `EntityIndexerRegistry` to allow specific indexes.

Reviewer: ```suggestion
* Added `EntityIndexerRegistry::EXTENSION_INDEXER_ONLY` to the `EntityIndexerRegistry` to allow specific indexers.

# Upgrade Information

## Add possibility to specify indexer in tontext
< Add some information and how to use it here>
```
